code_challenges/code_wars/create_phone_number.py

```diff
@@ -0,0 +1,6 @@
+# https://www.codewars.com/kata/525f50e3b73515a6db000b83
+
+
+def phone_format(numbers):
+    numbers = "".join(str(i) for i in numbers)
```

Comment:
Nice.

`str.join` https://docs.python.org/3/library/stdtypes.html#str.join takes an iterable.  But you are passing: `str(i) for i in numbers` which is not a [list comprehension ](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions).  It's missing the `[` and `]` to make it an actual list.  What is this called?

<details><summary>Click For Answer</summary>
<a href="https://peps.python.org/pep-0289/">a generator expression</a>
</details>